rpc/Base.ts

```diff
@@ -18,10 +18,10 @@ export type Subscription<NotificationResult = any> = { [_N]: NotificationResult
 
 export interface ClientProps<
   M extends AnyMethods,
-  Beacon,
+  DiscoveryValue,
```

Comment:
Zeroing in on correct terminology. Beacons contain discovery values (such as URLs and chain specs). Beacons are encoded with the type information necessary to constrain client usage.